### DISCUSSION TITLE
Error 404 on projects administration page

### DISCUSSION THREAD

**Discussion by pcxeon2** [kimai/kimai#4208] [On-Premise: Installation & Updates]
### Describe the issue

Good morning,

I have been using kimai for over a year and update it regularly.
Debian 11, Apache/2.4.56, php 8.2.8

Since I applied the 2.0.29 update this morning, kimai returns me a 404 error when I go to Administration > Projects.
All other pages are working fine.
It was working before on 2.0.28

The apache and kimai logs do not return any errors.

Thank you for your assistance.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.0.29

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.2

### Logfile

/var/log/prod.log doesnt exist
Dactor > logs > Empty logfile

### Screenshots

![Screenshot 2023-07-27 104559](https://github.com/kimai/kimai/assets/20686533/3251954a-d476-4a33-a8ce-6a1b1802ee0f)

**Comment by kevinpapst** [comment]
I converted this to a discussion, because it is not a bug.

I understand your frustration, but nothing between 2.0.28 and 2.0.29 is able to cause the behavior:
https://github.com/kimai/kimai/compare/2.0.28...refs/heads/main

There are thousands of automated tests, that would find such a problem. 
The demo works as well. 
So this is some kind of environment related issue that I cannot see and therefor cannot really help with.

Your own server or self hosted?
There are known issues when using Kimai in a PHP environment that runs ioncubeloader.
